src/web/routes/application/child-date-of-birth/child-date-of-birth.js

```diff
@@ -50,5 +50,5 @@ const childrenDob = {
 module.exports = {
   behaviourForGet,
   behaviourForPost,
-  childrenDob
+  childDateOfBirth: childDateOfBirth
```

Comment:
This can use shorthand syntax (as previous):
```
{
  ...
  childDateOfBirth
}
```

Reply:
Sorry - thought I'd caught all of those